tests/Functional/FunctionalTestCase.php

```
@@ -127,6 +127,40 @@ protected function getPhpcsViolations(string $output): array {
+   *   The sniff to run (e.g., 'DrevOps.NamingConventions.LocalVariableNaming').
+   *
+   * @return string
+   *   The fixed file content.
+   */
+  protected function runPhpcbf(string $file_path, string $sniff): string {
+    $phpcbf_bin = __DIR__ . '/../../vendor/bin/phpcbf';
+    $this->assertFileExists($phpcbf_bin, 'PHPCBF binary must exist');
+    $this->assertFileExists($file_path, 'File to fix must exist');
+
+    // Copy fixture to a temp file so we don't modify the original.
+    $temp_file = tempnam(sys_get_temp_dir(), 'phpcbf_test_');
```

PHPMD (2.15.0)

[error] The variable $temp_file is not named in camelCase. (undefined)

(CamelCaseVariableName)